plugins/obs-filters/chroma-key-filter.c

```diff
@@ -24,6 +25,13 @@
 #define TEXT_SIMILARITY                obs_module_text("Similarity")
 #define TEXT_SMOOTHNESS                obs_module_text("Smoothness")
 #define TEXT_SPILL                     obs_module_text("ColorSpillReduction")
+#define TEXT_SPILL_TYPE                obs_module_text("ColorSpillReductionType")
+
+enum SpillType {
```

Comment:
For C code we use snake case for names. This should be defined as enum spill_type